notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<ul><li>Same comment: The <code>tf.keras.layers</code> plus formatting make this code somehow not easy on the eye</li><li>Add a explanation paragraph above explaining what this function does and why we need it</li><li>Generally, I'd remove the function comments and instead explain what they do in a markdown cell above the function. It feels the code in this lab is unnecessary long and verbose</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I removed `tf.keras.layers` from the lab.
And removed helper functions. Thanks!